src/icons/languageIcons.ts

```diff
@@ -76,4 +76,6 @@ export const languageIcons: LanguageIcon[] = [
     { icon: { name: 'vim' }, ids: ['viml'] },
     { icon: { name: 'silverstripe' }, ids: [] },
     { icon: { name: 'prolog' }, ids: ['prolog'] },
+    { icon: { name: 'reason' }, ids: ['reason', 'reason_lisp'] },
```

Comment:
What kind of purpose has this addition of reason language id here? What is reason_lisp? Correct me, but it hasn't something to do with ML right?

Reply:
It's an Ocaml extension language. `reason_lisp` is the ID used by [this extension](https://marketplace.visualstudio.com/items?itemName=jaredly.reason-vscode).

Reply:
Thank you, maybe you can put it into a separate pull request next time, it makes reviews easier. However, great additions and good pull request overall.